cmd/check.go

```diff
@@ -73,6 +74,10 @@ and/or users.`,
 				AllInstallModes:        checkflags.AllInstallModes,
 			}
 
+			if checkflags.ExtraCRDirectory != "" {
+				capAuditor.ExtraCRDirectory(checkflags.ExtraCRDirectory)
```

Comment:
Add an error return to ExtraCRDirectory and handle appropriately here. Should it continue if there is an error in the extras? Are they considered optional? If not, we should return an error.

Reply:
> If not, we should return an error

This makes me modify the cobra `Run` command to `RunE`.

This made me realize that `capAuditor.RunAudits` also returns an error, but we don't actually do anything with it. We probably also should return an error in `RunE`, right ? This feels like a change which is a bit out of scope, so I'll package it in another commit: 0c896e62c2dda9f2f34efb8ec69b794ea8408249